analyzer/util/util.go

```diff
@@ -31,7 +31,7 @@ type Backoff struct {
 
 // NewBackoff returns a new backoff.
 func NewBackoff(initialTimeout time.Duration, maximumTimeout time.Duration) (*Backoff, error) {
```

Comment:
could use some description, as we're finding this incorrect usage and not changing the API

Reply:
Added description after refactoring things a bit.